ank/src/cli_commands.rs

```diff
@@ -2129,26 +2086,18 @@ mod tests {
             runtime_config: test_workload_runtime_cfg.clone(),
             ..Default::default()
         };
-        let mut updated_state = startup_state.clone();
-        updated_state
+        let mut complete_state_update = CompleteState::default();
+        complete_state_update
             .desired_state
             .workloads
             .insert(test_workload_name.clone(), new_workload);
 
         let mut mock_client_builder = MockGRPCCommunicationClientBuilder::default();
-        mock_client_builder.expect_receive_request(
-            "first_complete_state_request",
-            RequestContent::CompleteStateRequest(CompleteStateRequest { field_mask: vec![] }),
-        );
-        mock_client_builder.will_send_response(
-            "first_complete_state_request",
-            ResponseContent::CompleteState(Box::new(startup_state.clone())),
-        );
         mock_client_builder.expect_receive_request(
             "update_state_request",
             RequestContent::UpdateStateRequest(Box::new(commands::UpdateStateRequest {
-                state: updated_state.clone(),
-                update_mask: vec!["desiredState".to_string()],
+                state: complete_state_update.clone(),
+                update_mask: vec!["desiredState.workloads.".to_string() + &test_workload_name],
             })),
         );
         mock_client_builder.will_send_response(
```

Comment:
This was already existing before your changes, but the `format!` call is needless here. As an alternative we can keep the format call here, but using the `test_workload_name` variable inside the format.

Reply:
I have refactored it.